MAINTAINERS.yaml

```diff
@@ -27,11 +26,18 @@
 #     maintainers:
 #       - John Doe <john.doe@gem5.org>
 #       - Jane Doe <jane.doe@gem5.org>
-#
+#     experts:
+#       - Jack Doe <jack.doe@gem5org>
+#       - Jill Doe <jill.doe@gem5org>
 #
 # The status field should have one of the following values:
 #   - maintained: The component has an active maintainer.
 #   - orphaned: The component is looking for a new owner.
+#
+# The experts field is optional and used to identify people who are
+# knowledgeable about the subsystem but are not responsible for it. Those
+# listed as an expert are typically good to add as a reviewer for pull requests
+# targeting that subsystem.
 
 
 pmc:
```

Comment:
Let's drop the PMC here. I would rather have this list outside the main repo. Maybe point to the maintainers team so we have a little more agility with this list. https://github.com/orgs/gem5/teams/maintainers

Reply:
I'm not sure if you can see the team https://github.com/orgs/gem5/teams/maintainers unless you're a member of the gem5 organization. At least when i logout of GitHub and try to go there I don't see anything. I couldn't find any way to make it public. Given that I've opted just to remove this.